src/components/pill/pill.spec.tsx

```diff
@@ -1,30 +1,37 @@
 import React from "react";
-import TestRenderer from "react-test-renderer";
-import { shallow, mount } from "enzyme";
+import { shallow, mount, ReactWrapper, ShallowWrapper } from "enzyme";
 import { shade } from "polished";
 
-import Pill from "./pill.component";
+import Pill, { PillProps } from "./pill.component";
+import StyledPill from "./pill.style";
 import styleConfig from "./pill.style.config";
 import { rootTagTest } from "../../__internal__/utils/helpers/tags/tags-specs";
 import {
   assertStyleMatch,
-  carbonThemesJestTable,
   testStyledSystemMargin,
   expectConsoleOutput as expectError,
 } from "../../__spec_helper__/test-utils";
 import IconButton from "../icon-button";
-import { baseTheme } from "../../style/themes";
+import {
+  aegeanTheme,
+  baseTheme,
+  mintTheme,
+  sageTheme,
+} from "../../style/themes";
 import { toColor } from "../../style/utils/color";
 
-const modernStyleTypes = ["neutral", "negative", "positive", "warning"];
+const modernStyleTypes = [
+  "neutral",
+  "negative",
+  "positive",
+  "warning",
+] as const;
 
 describe("Pill", () => {
-  const render = (props, renderer = mount) => {
-    return renderer(<Pill {...props}>My Text</Pill>);
-  };
+  const render = (props?: PillProps) => <Pill {...props}>My Text</Pill>;
```

Comment:
```suggestion
  const render = (props?: PillProps, renderer = mount) => renderer(<Pill {...props}>My Text</Pill>);
```
In my opinion this function is a bit misleading since its not rendering anything when called but returning the component. What do you think?

Reply:
I get what your saying, I think I'll leave the function implemented as is, as it won't need refactoring when we migrate to RTL, but I'll rename it so it's a little clearer what it's doing